### PR TITLE
39 Prepare readme file before demo

### LINKED ISSUE #39: Prepare MVP demo
Can be presentation or any other form

- [x] out objectives (mainly for self-development in SwiftUi that is used in DT app)
- [x] our approach to collaboration and work prioritisation
- [x] tech stack and why like that
- [x] how we benefitted from this project
- [ ] show kanban with all features (done/future)
- [ ] define happy flow
- [ ] prepare data set
- [ ] add demo presentation to Readme
- [ ] share repo

### PR DESCRIPTION
**Pull request by ivansdivans**
## 🧙🏻‍♂️ Describe your changes

- Added introduction to app
- Added learning from this pet project
- Added info about currently supported features

## 🔗 Issue that is implemented here 

- Closes #39

## 📸 Screenshot or screen recording of working solution

|   #   | File  |
| ----- | ----- |
|   1   | see readme page |

## 👮🏻‍♂️ Check yourself

- [ ] I have performed a self-review of my code
- [x] I have resolved all TODOs in this branch
- [x] Changes are covered by unit tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,48 @@
...
+All the above mentioned directly benefited client's project where SwiftUI was being introduced for the first time.
+
+## Project status
+
+Currently project is in MVP stage which includes following features:
+- Creating and configuring your household (including name, sunlight level, air quality, humidity and average temperature)
+- Adding up to 5 plants from your phone's gallery
+- Automatically clasifying plant using trained ML model
+- Editing plant's details
+- Generating plants care schedule based on watering frequency
```
Comment (Inn0) [inline README.md:34]:
```suggestion
- Generating a household's care schedule based on watering frequency of selected plants
```

File: README.md [hunk 2]
```diff
@@ -1,10 +1,48 @@
...
+All features and ideas for future development are summarised in [Kanban board](https://github.com/users/ivansdivans/projects/2).
+
+This project has [Wiki](https://github.com/ivansdivans/bloom-buddy-mobile/wiki) with working materials (inlcuding brainstorming session notes, wireframes and requirements). 
 
 ## Technologies used
 1. [XCode](https://developer.apple.com/xcode/) - Integrated Development Environment
 2. [Swift](https://developer.apple.com/swift/) - Programming Language
 3. [SwiftUI](https://developer.apple.com/xcode/swiftui/) - User Interface Framework
 4. [CoreML](https://developer.apple.com/documentation/coreml) - Machine Learning Framework
+5. [Kaggle](https://www.kaggle.com/) - plants and flowers datasets
```
[the remark this answers is not in the document]
Comment (ivansdivans) [reply README.md:45]: Yeah, I was doubting this point myself. Will remove it from technologies section.

File: README.md [hunk 3]
```diff
@@ -13,7 +51,5 @@
 5. Development conventions - keep `main` ready-to-deploy at all times; use [feature-flags](https://martinfowler.com/articles/feature-toggles.html) for potentially breaking changes
 6. Testing strategy - meaningful unit tests for each feature and bugfix
 7. Merging policy - mandatory PR, build and unit tests are green, 1 approval; no force pushes
-
-## Credits
-
-This pet project is created by [Daan Brocatus](https://github.com/Inn0) and [Ivans Mihailovs](https://github.com/ivansdivans). Their intention is to practice SwiftUI and try innovative solutions. 
+8. Feature tracking - each feature/bug is registered as an issue in Github and added to project (Kanban board)
+9. Feature prioritisation - issues are priritised based on trafic lights system; focus on mvp related features, highest priority & smallest in size are implemented first
```
[the remark this answers is not in the document]
Comment (ivansdivans) [reply README.md:55]: Agree, will add

**Review by Inn0 — APPROVED** [review]
Happy to see that you've implemented my feedback very thoroughly 😄 LGTM 🚀

**Pull request merged**